pkg/packetfilter/nftables/namedset.go

```diff
@@ -35,6 +36,9 @@ type namedSet struct {
 
 func (p *packetFilter) NewNamedSet(set *packetfilter.SetInfo) packetfilter.NamedSet {
 	setType := "ipv4_addr"
+	if p.family == k8snet.IPv6 {
+		setType = "ipv6_addr"
```

Comment:
@yboaron I assume this is correct for IPv6? I couldn't find any concrete info/doc to confirm it...